sanity-io/structured-content-2022
# add ticket groups

# Add ticket groups

## Intent

Support grouping ticket price availabilities tied to venues for multidimensional tables.

## Description

This PR introduces a change to an existing data model, and hence data needs to migrated. 


## Testing the frontend changes

1. Open [/registration-info](https://structured-content-2022-web-git-add-ticket-groups.sanity.build/registration-info)
2. Scroll down to the table under the "Ticket details" heading and verify that it renders in accordance with the [corresponding Figma sketches](https://www.figma.com/file/XjAvJKsLpxYJSc8QDWK81j/Sanity-SCC-Website-(Shared)?node-id=641%3A4349) for all supported viewports
